### PR TITLE
feat: add keyGet and keyGet2 functions.

### PR DESCRIPTION
**Pull request by shy1st**
Fix: #132

### CODE REVIEW

File: src/test/java/org/casbin/jcasbin/main/BuiltInFunctionsUnitTest.java [hunk 1]
```diff
@@ -61,4 +63,53 @@ public void testGlobMatchFunc() {
...
+        testKeyGet2("/foo/bar", "/foo/*", "id","");
+        testKeyGet2("/foobar", "/foo", "id","");
+        // different with KeyMatch.
+        testKeyGet2("/foobar", "/foo*", "id","");
+        testKeyGet2("/foobar", "/foo/*", "id","");
+
+        testKeyGet2("/", "/:resource", "resource","");
+        testKeyGet2("/resource1", "/:resource", "resource","resource1");
+        testKeyGet2("/myid", "/:id/using/:resId", "id","");
+        testKeyGet2("/myid/using/myresid", "/:id/using/:resId", "id","myid");
```
Comment (hsluoyz) [inline src/test/java/org/casbin/jcasbin/main/BuiltInFunctionsUnitTest.java:97]: code format.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/test/java/org/casbin/jcasbin/main/BuiltInFunctionsUnitTest.java (hunk 1 above)
Comment (shy1st) [reply src/test/java/org/casbin/jcasbin/main/BuiltInFunctionsUnitTest.java:97]: Sorry, I don't know what is the problem of this code format.

Comment (hsluoyz) [reply src/test/java/org/casbin/jcasbin/main/BuiltInFunctionsUnitTest.java:97]: A space after comma (,), many lines have such problem.

**Comment by hsluoyz** [comment]
@shy1st can we enable the semantic-pr bot?

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**